### PR TITLE
REF: Move from metaclass Backend to base class.

### PR DESCRIPTION
**Pull request by hhslepicka**
## Description
Switching to a base class as in this case of the backend there is no particular reason (as far as I could tell) for it to be a metaclass.

## How Has This Been Tested?
Tests are passing.

### CODE REVIEW

**Comment by klauer** [comment]
flake8 failing :(

File: happi/backends/core.py [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
-class Backend(abc.ABCMeta):
+class _Backend:
     """
-    Abstract interface for backend database
+    Base class for backend database
     """
-    @abc.abstractproperty
-    def devices(self):
+
+    def all_devices(self):
```
Comment (klauer) [inline happi/backends/core.py]: Let's keep this a `@property` otherwise it's callable now instead

Comment (hhslepicka) [reply happi/backends/core.py]: It was deleted by accident. Addressed via https://github.com/pcdshub/happi/pull/115/commits/e44f98f7a5daeac13ef52c1a07c66d907260eb2e

**Comment by ZLLentz** [comment]
This looks good to me as well a day later

**Pull request merged**